@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -37,4 +37,10 @@ export default {
       labelSuffix: "delete",
     },
   },
+  Table: {
```

Comment:
Something we might not have discussed is how "generic" some of these translations should be, and i guess `expand` and `collapse` is one of the main ones. We will have this translation for i think: accordion, readmore, formprogress, expansioncard, table. 

Should we have a "generic" key for "expand":
```tsx
Expansion: {
      expand: "Show more",
      collapse: "Show less",
    },
```

`formprogress` complicates this a little since it's a "visible" label, but does it matter 🤔 ? Both regular and SR-users  should experience the same application.